src/utils/constants.ts

```diff
@@ -280,5 +280,38 @@ export const FORMATTED_SOUL_WARS = 'Soul Wars Zeal';
 export const FORMATTED_LEAGUE_POINTS = 'League Points';
 export const FORMATTED_RIFTS_CLOSED = 'Rifts closed';
 
-export const INVALID_FORMAT_ERROR = 'Invalid hiscores format';
-export const PLAYER_NOT_FOUND_ERROR = 'Player not found';
+export class InvalidFormatError extends Error {
+  __proto__ = Error;
+
+  constructor() {
+    super('Invalid hiscores format');
+    Object.setPrototypeOf(this, InvalidFormatError.prototype);
+  }
+}
+
+export class InvalidRSNError extends Error {
+  __proto__ = Error;
+
+  constructor(message: string) {
+    super(message);
+    Object.setPrototypeOf(this, InvalidRSNError.prototype);
```

Comment:
This prototype setting seems redundant. What's the purpose for our use case?

Reply:
Without `Object.setPrototypeOf(this, InvalidRSNError.prototype);` then `(new InvalidRSNError('test')) instanceof InvalidRSNError` will return false.

I took it directly from [here](https://github.com/microsoft/TypeScript-wiki/blob/81fe7b91664de43c02ea209492ec1cea7f3661d0/Breaking-Changes.md#extending-built-ins-like-error-array-and-map-may-no-longer-work).

A more elegant solution to this would be to use something like `ts-custom-error`, but I did not want to add dependencies if I could avoid it.